### PR TITLE
WEBEU2 - Preprocessing I - Rui Hildt 🇧🇪

### PR DESCRIPTION
**Pull request by ruihildt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,40 +4,40 @@ For this project you will be building a digital resume from scratch. You will be
 
 ## Task 1: Set Up The Project With Git
 
-- [ ] Create a forked copy of this project.
-- [ ] Add your project manager as collaborator on Github.
-- [ ] Clone your OWN version of the repository (Not Lambda's by mistake!).
-- [ ] Create a new branch: git checkout -b `<firstName-lastName>`.
-- [ ] Implement the project on your newly created `<firstName-lastName>` branch, committing changes regularly.
-- [ ] Push commits: git push origin `<firstName-lastName>`.
+- [x] Create a forked copy of this project.
```
Comment (maximesalomon) [inline README.md]: Great job, continue using the README like this!

File: index.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
 <head>
   <meta charset="utf-8">
 
-  <title>My Resume</title>
+  <title>Hodor Resume</title>
```
Comment (maximesalomon) [inline index.html]: Where is the `<meta name="viewport" content="width=device-width, initial-scale=1.0">`? :)

File: index.html [hunk 3]
```diff
@@ -14,6 +14,93 @@
...
+            <p>Hodor!</p>
+          </div>
+        </div>
+
+      </section>
+  
+    </main>
+
+  </div>
+
```
Comment (maximesalomon) [inline index.html]: Great job with the HTML overall. You might want to run Prettier through it however 💅

File: less/index.less [hunk 4]
```diff
@@ -1,59 +1,179 @@
...
-
-/* Set every element's box-sizing to border-box */
-* {
-    box-sizing: border-box;
-}
-
-html, body {
-    height: 100%;
+// imports
+@import "reset.less";
```
Comment (maximesalomon) [inline less/index.less]: Great job creating an other LESS file and importing it!

File: less/index.less [hunk 5]
```diff
@@ -1,59 +1,179 @@
...
+@import url('https://fonts.googleapis.com/css?family=ZCOOL+KuaiLe|Crimson+Text');
+
+// Breakpoints
+@mobileBreakpoint:360px;
```
Comment (maximesalomon) [inline less/index.less]: You could create a separate LESS file for the variables :)

File: less/index.less [hunk 6]
```diff
@@ -1,59 +1,179 @@
...
+
+// colors
+
+@grey: grey;
+@lightGray: rgb(218, 218, 218);
+@black: black;
+@white: white;
+
+// headers
+.header1() {
```
Comment (maximesalomon) [inline less/index.less]: Great job with the mixins!

File: less/index.less [hunk 7]
```diff
@@ -1,59 +1,179 @@
...
+				border: 0.2rem @grey solid;
+				border-top:none;
+				margin: 0;
+			}
+
+			p.list:nth-of-type(1) {
+				border-top: 0.2rem @grey solid;
+			}
+	}
+}
```
Comment (maximesalomon) [inline less/index.less]: No media queries?

File: index.html (hunk 2 above)
Comment (ruihildt) [reply index.html]: I'm already used to have that generated with emmet `!` command. Just added to todays's project. ;)

File: index.html (hunk 3 above)
Comment (ruihildt) [reply index.html]: Which prettier do you recommend? There's so many in VS Code extensions repo.

File: less/index.less (hunk 5 above)
Comment (ruihildt) [reply less/index.less]: Noted.

File: less/index.less (hunk 7 above)
Comment (ruihildt) [reply less/index.less]: In today's project.

**Pull request merged**